File: backend/app/Models/Event.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Event extends Model
{
    use HasFactory;

    protected $fillable = [
        'name',
        'organizer',
        'location',
        'price',
        'admin_id',
        'image_url',
        'description',
        'date',
        'time',
        'city',
    ];

    public function admin()
    {
        return $this->belongsTo(User::class, 'admin_id');
    }
}

File: android/app/src/main/java/com/example/dinus/model/Event.kt
package com.example.dinus.model

data class Event(
    val id: Int = 0,
    val name: String = "",
    val organizer: String = "",
    val location: String = "",
    val price: String = "",
    val admin_id: Int = 0,
    val image_url: String = "",
    val description: String = "",
    val date: String = "",
    val time: String = "",
    val city: String = "",
    val admin: User = User()
)

